src/main/kotlin/Commands.kt

```diff
@@ -34,6 +34,7 @@ val acceptHostingCommand = command("Да", "acceptHosting", 3)
 val declineHostingCommand = command("Нет", "declineHosting", 3)
 
 val menuLobbyCommand = command("◀️ Меню игроков", "menuLobby", 1)
+val toggleHideRolesModeCommand = command("🙈 Скрывать роли", "toggleHideRolesMode", 1)
```

Comment:
All commands are grouped by domain of usage, this one is out of place in between menu buttons